open-craft/sprints
# [BB-1622] Prepare API for accounts

This introduces the API and frontend for the accounts dashboard.
The accounts dashboard is aware of the sprint board's current view (whether it's showing cells/cell's board/person's board). Therefore, when you click on the cell's name, the sustainability dashboard recalculates its data for displaying cell/person-related data only.

Half of calculations are performed on the backend and the other half on the frontend side and stored in the Local Storage, so the navigation is smooth. To reload the accounts dashboard you need to refresh the whole page.

## Testing instructions:
1. Set `.env` file (you can take this from BB-1621).
1. Start the backend with `docker-compose -f local.yml up --build`.
1. Create frontend/.env.local and set REACT_APP_GOOGLE_CLIENT_ID there.
1. Load the fixtures (attached to the ticket) with `docker-compose -f local.yml exec django python manage.py loaddata sprints/fixtures/fixtures_sprints.json`.
1. Navigate to frontend and run npm i && npm start.
1. Log in.
1. Check the Budget dashboard for overall/cells/yourself and see if the accounts look fine.
